crates/bevy_pbr/src/render/light.rs

```diff
@@ -1635,7 +1636,7 @@ pub fn queue_shadows<M: Material>(
             // NOTE: Lights with shadow mapping disabled will have no visible entities
             // so no meshes will be queued
             for entity in visible_entities.iter().copied() {
-                let Some(mesh_instance) = render_mesh_instances.get(&entity) else {
+                let Some(mesh_instance) = render_mesh_instances.get_mut(&entity) else {
```

Comment:
This doesn't look like it needs to be `mut`?

Reply:
it's modified @1689 to add the shadow_bind_group_id

Reply:
Ah, not sure how I missed that.